util/config/compileline

```diff
@@ -1,154 +1,186 @@
-    print "[VAR=VALUE] sets any environment variable\n";
-    print "[options] include\n";
-    print " --llvm specify that the rest of the arguments are for a compile\n";
-    print "        working with the LLVM backend. This argument causes\n";
-    print "        this script to set CHPL_TARGET_COMPILER=clang-included\n";
-    print " --home  print out CHPL_HOME\n";
-    print " --make  print out the make command\n";
-    print " --compile  print out the command to compile a file including\n";
-    print "            Chapel runtime headers\n";
-    print " --includes-and-defines print out any -D and -I options to\n";
-    print "                        include the Chapel runtime\n";
-    print " --libraries print out any -L and -l options to\n";
-    print "             link with the Chapel runtime\n";
-    print " --main.o print out the path to the main.o file\n";
-    print " --clang print out the path to clang and clang++\n";
-    print " --clang-sysroot-arguments print out any saved clang arguments\n";
-    print "                           that specify the system root\n";
-  }
-}
+  ret = ""
+  output = subprocess.check_output(cmd, shell=True)
+  lines = output.splitlines()
+  for line in lines:
+    if not line.startswith("make"):
+      ret += line
+
+  return ret
+
+def mysystem(cmd):
+
+  s = myrun(cmd)
+  sys.stdout.write(s)
+
+
+def _main():
+
+  script_dir = os.path.dirname(os.path.realpath(__file__))
+  chpl_home_dir = os.path.normpath(script_dir + "/../../");
+  check_file = os.path.relpath(__file__, chpl_home_dir)
+  check_path = chpl_home_dir + "/" + check_file;
+  if not os.path.isfile(check_path):
+    sys.stderr.write("Warning: check {0} not found\n".format(check_path));
+
+  if "CHPL_HOME" in os.environ:
+    if os.path.abspath(os.environ["CHPL_HOME"]) != chpl_home_dir:
+      # to be sure, check that the inode numbers of our check file match
+      env_check = os.environ["CHPL_HOME"] + "/" + check_file
+      rel_check = chpl_home_dir + "/" + check_file
+      if os.path.samefile(env_check, rel_check):
+        # No warning, it's OK, they are the same file.
+        pass
+      else:
+        sys.stderr.write("Warning: check {0} not found\n".format(check_path));
+        sys.stderr.write("Warning: Mismatched CHPL_HOME; got {0} but expected {1}\n".format(os.path.abspath(os.environ["CHPL_HOME"]), chpl_home_dir));
+    chpl_home_dir = os.environ["CHPL_HOME"] # use enviro var spelling of it
+  else:
+    os.environ["CHPL_HOME"] = chpl_home_dir
+
+  os.environ["CHPL_MAKE_HOME"] = chpl_home_dir;
+  if "CHPL_RUNTIME_LIB" in os.environ:
+    os.environ["CHPL_MAKE_RUNTIME_LIB"] = os.environ["CHPL_RUNTIME_LIB"]
+
+  if "CHPL_RUNTIME_INCL" in os.environ:
+    os.environ["CHPL_MAKE_RUNTIME_INCL"] = os.environ["CHPL_RUNTIME_INCL"]
+
+  if "CHPL_THIRD_PARTY" in os.environ:
+    os.environ["CHPL_MAKE_THIRD_PARTY"] = os.environ["CHPL_THIRD_PARTY"]
+
+  make = subprocess.check_output([chpl_home_dir +  "/util/chplenv/chpl_make.py"])
```

Comment:
Can/should probably just import chpl_make and get the value directly, instead of shelling out